src/routes/Board/Board.tsx

```diff
@@ -32,15 +32,16 @@ function Board(props: BoardProps) {
         return <LoadingScreen/>;
     } else if (state.board.status === 'ready') {
         return (
-            <BoardComponent>
-                {
-                    state.board.data!.columns.map((column) => (<Column color="pink">{column.name}</Column>))
-                }
-            </BoardComponent>);
-        
+            <>        
+                <BoardComponent>
+                        {state.board.data!.columns.map((column:any) => (
+                            <Column color="pink" key={column.id} columnId={column.id}/>
```

Comment:
We should move the `color` prop inside of the `<Column />` component. It can access the set color value by the provided id prop.